src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 30px 10px;
+  width: $calendar-width;
+
+  list-style-type: none;
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
It's the general rule that will be applied to the all elements inside the project, will be better to set it using `*` selector

Reply:
Its not, if talk about box-sizing

Reply:
Agree, as it's how it works for now. But generally, when you'll use it in the large projects, you'll specify it as the general rule for all elements, as you should decide how you'll calculate the size for the all elements
So this property should work the same for all elements on your page